.github/workflows/vitest.yml

```
@@ -9,37 +9,40 @@ on:
+      - name: Generate Panda CSS
+        run: pnpm exec panda codegen && pnpm exec panda cssgen --clean
+
+      - name: Run Vitest
+        run: npx vitest --coverage.enabled true
+
+      - name: Report Coverage
+        if: always()
+        uses: davelosert/vitest-coverage-report-action@v2
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)